### PR TITLE
cohen bad

### PR DESCRIPTION
**Pull request by Hunter-The-Great**

### CODE REVIEW

**Comment by drew-harris** [comment]
Not approved, formatting does not follow default prettier spec. Please fix

File: src/ai.ts [hunk 1]
```diff
@@ -7,59 +7,61 @@ const defaultScript = "You are a helpful discord chatbot";
 const API_URL = "https://api.fireworks.ai/inference/v1";
 
 const getScript = async (): Promise<string> => {
-  const scriptResult = await db
-    .select()
-    .from(scripts)
-    .where(eq(scripts.isActive, true))
-    .execute();
+    const scriptResult = await db
```
Comment (drew-harris) [inline src/ai.ts:10]: we dont use 4 spaces

Comment (cssmith8) [reply src/ai.ts:10]: 🗣️ 🔥 ‼️

Comment (Hunter-The-Great) [reply src/ai.ts:10]: time for war

**Pull request closed without merging**